sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -448,9 +452,18 @@ export class EventHubConsumer {
         ? retryOptions.retryInterval / 1000
         : Constants.defaultDelayBetweenOperationRetriesInSeconds;
 
+    const timeoutInMs = retryOptions.timeoutInMs;
+    const retryPolicy = retryOptions.retryPolicy;
```

Comment:
We stated that `retryPolicy` will default to `LinearRetryPolicy` in the docs. This would be a good place to set it if it is currently undefined.